pom.xml

```diff
@@ -59,7 +59,6 @@
     <json.version>20210307</json.version>
     <jackson.version>2.12.2</jackson.version>
     <jackson-databind.version>2.12.2</jackson-databind.version>
-    <legacy-jackson.version>1.9.12</legacy-jackson.version>
```

Comment:
I couldn't find any other consumes of this prop and the version mentioned here matches the Jackson version included by the old ESRI implementation exactly.